Assets/Scripts/InGame/Betting/Betting.cs

```diff
@@ -187,6 +194,8 @@ public void Bet(NetworkPlayer player, BetActionInfo obj)
                  _raiseCount++;
                  break;
              case BetAction.AllIn:
+                 // proc all in? 
```

Comment:
@Created-for-a-purpose assuming here we just have to consider raise action with amount equals whatever the player has